bin/gt.js

```diff
@@ -104,6 +110,7 @@ async function main() {
 	const proxy = new Proxy({
 		ip: argv['ip'],
 		port: parseInt(argv['port'], 10),
+		httpsOnly: argv['httpsOnly'],
```

Comment:
`httpsOnly` or `https-only`?

Reply:
Yea you right, `https-only` is correct, But why it works with this problem?
idk whats happened, but fixed now.